src/test/fixtures/generator.mjs

```diff
@@ -0,0 +1,138 @@
+import { configFileV2 } from "./configuration-file.mjs";
+import { ropsten, rinkeby, did } from "./credential.mjs";
+import { existsSync, writeFileSync, mkdirSync } from "fs";
+import { fileURLToPath } from "url";
+import { join, dirname, parse } from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const V2_ROPSTEN_DIRECTORY = join(__dirname, "_generated", "v2", "config", "ropsten", "sample-config-ropsten.json");
+const V2_AWS_ROPSTEN_DIRECTORY = join(
+  __dirname,
+  "_generated",
+  "v2",
+  "config",
+  "ropsten",
+  "sample-config-ropsten-aws.json"
+);
+const V2_RINKEBY_DIRECTORY = join(__dirname, "_generated", "v2", "config", "rinkeby", "sample-config-rinkeby.json");
+const V2_DID_DIRECTORY = join(__dirname, "_generated", "v2", "config", "did", "sample-config-did.json");
+
+const V3_ROPSTEN_DIRECTORY = join(__dirname, "_generated", "v3", "config", "ropsten", "sample-config-ropsten.json");
+const V3_RINKEBY_DIRECTORY = join(__dirname, "_generated", "v3", "config", "rinkeby", "sample-config-rinkeby.json");
+const V3_DID_DIRECTORY = join(__dirname, "_generated", "v3", "config", "did", "sample-config-did.json");
+
+function generateConfigFile() {
+  //Ropsten
+  const ropstenConfigFile = JSON.parse(JSON.stringify(configFileV2));
+  ropstenConfigFile.network = ropsten.network;
+  ropstenConfigFile.wallet = ropsten.wallet;
+  ropstenConfigFile.documentStorage.url = ropsten.documentStorageUrl;
+  const ropstenConfigFileV2 = insertV2Credential(ropstenConfigFile, ropsten);
+  updateConfigFile(ropstenConfigFileV2, V2_ROPSTEN_DIRECTORY);
+
+  //AWS Ropsten
+  const ropstenAWSConfigFile = JSON.parse(JSON.stringify(configFileV2));
+  ropstenAWSConfigFile.wallet = ropsten.walletAWS;
+  updateConfigFile(ropstenAWSConfigFile, V2_AWS_ROPSTEN_DIRECTORY);
+
+  //Rinkeby
+  const rinkebyConfigFile = JSON.parse(JSON.stringify(configFileV2));
+  rinkebyConfigFile.network = rinkeby.network;
+  rinkebyConfigFile.wallet = rinkeby.wallet;
+  rinkebyConfigFile.documentStorage.url = rinkeby.documentStorageUrl;
+  insertV2Credential(rinkebyConfigFile, rinkeby);
+  updateConfigFile(rinkebyConfigFile, V2_RINKEBY_DIRECTORY);
+
+  //DID
+  const didConfigFile = JSON.parse(JSON.stringify(configFileV2));
+  didConfigFile.network = did.network;
+  didConfigFile.wallet = did.wallet;
+  delete didConfigFile.documentStorage;
+  insertV2Credential(didConfigFile, did, true);
+  updateConfigFile(didConfigFile, V2_DID_DIRECTORY);
+}
+
+function insertV2Credential(configFile, credential, removeTransferableRecord = false) {
+  configFile.forms.forEach((form, index, object) => {
```

Comment:
this is weird, lemme think

Reply:
use map instead

1. export `forms` as array, something like:
```
import { forms as documentFormsV2 } from "./forms/v2/index.mjs";
```
2. map and return updated form:
```
  const { network, wallet, documentStorageUrl, documentStore, tokenRegistry } = ropsten;

  const forms = documentFormsV2.map((form) => {
    switch (form.type) {
      case "TRANSFERABLE_RECORD":
        return {
          ...form,
          defaults: {
            ...form.defaults,
            issuers: [tokenRegistry],
          },
        };
      case "VERIFIABLE_DOCUMENT":
        return {
          ...form,
          defaults: {
            ...form.defaults,
            issuers: [documentStore],
          },
        };
      default:
        throw Error("no form type found");
    }
  });
  ```
3. consume the `forms`
```
  const ropstenConfigFile = {
    ...configFileV2,
    network,
    wallet,
    documentStorageUrl,
    forms,
  };
  ```

Reply:
done.